scripts/manage_root_dir_config.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+"""
+Add, modify or remove value for root_dir in config file.
+root_dir
+"""
+import os
+from argparse import ArgumentParser, RawTextHelpFormatter
+from GCRCatalogs import write_to_user_config, remove_root_dir_default
```

Comment:
`write_to_user_config` no longer exists I think
```suggestion
from GCRCatalogs import set_root_dir, remove_root_dir_default
```